test/integration/multiplexed_upstream_integration_test.cc

```diff
@@ -361,25 +362,18 @@ TEST_P(MultiplexedUpstreamIntegrationTest, ManyLargeSimultaneousRequestWithBuffe
   manySimultaneousRequests(1024 * 20, 1024 * 20);
 }
 
-TEST_P(MultiplexedUpstreamIntegrationTest, ManyLargeSimultaneousRequestWithRandomBackup) {
+TEST_P(MultiplexedUpstreamIntegrationTest, DISABLED_ManyLargeSimultaneousRequestWithRandomBackup) {
   // random-pause-filter does not support HTTP3.
   if (upstreamProtocol() == Http::CodecType::HTTP3) {
     return;
   }
 
-  if (GetParam().defer_processing_backedup_streams) {
-    // TODO(kbaichoo): fix this test to work with deferred processing by using a
```

Comment:
I think the comment should be tagged next to the test name - to keep the context around why the test was disabled (consider adding a tracking issue for this).

Reply:
good idea, done.